ompi/errhandler/errcode.c

```diff
@@ -241,6 +242,7 @@ int ompi_mpi_errcode_init (void)
     CONSTRUCT_ERRCODE( ompi_err_proc_fail_pending,  MPI_ERR_PROC_FAILED_PENDING,  "MPI_ERR_PROC_FAILED_PENDING: Process Failure during an MPI_ANY_SOURCE non-blocking receive, request is still active" );
     CONSTRUCT_ERRCODE( ompi_err_revoked,  MPI_ERR_REVOKED,  "MPI_ERR_REVOKED: Communication Object Revoked" );
 #endif
+    CONSTRUCT_ERRCODE( ompi_err_session,  MPI_ERR_SESSION,  "MPI_ERR_SESSION: Invalid session handle" );
```

Comment:
I don't know if this is a problem, but looking at the pattern for the other error codes, I see a use in an OBJ_DESTRUCT macro, but I don't see one for this new error code

Reply:
As per [this comment](https://github.com/open-mpi/ompi/pull/10589#issuecomment-1190335579), actually using the new error code is in @hppritcha TODO list. Besides using it, having the error code implemented with its error string and defined in `mpi.h` is a requirement for MPI-4 codes to compile if these codes use  `MPI_ERR_SESSION`.

Reply:
good catch @drwootton fixing.

Reply:
@drwootton check now, thanks

Reply:
looks ok now